src/Writing/PostmanCollectionWriter.php

```diff
@@ -183,6 +183,14 @@ protected function makeUrlData($route)
             })->values()->toArray(),
         ];
 
+        // Create raw url-parameter (Insomnia uses this on import)
+        $query = collect($base['query'] ?? [])->map(function ($array) {
```

Comment:
Let's be a little more semantic in our naming. Let\s rename this from `$array` to something more descriptive, like `$queryParamData`.